tests/gold_tests/timeout/active_timeout.test.py

```diff
@@ -60,3 +60,8 @@
 tr3= Test.AddTestRun("tr")
 tr3.Processes.Default.Command = 'curl -k -i --http2 https://127.0.0.1:{0}/file'.format(ts.Variables.ssl_port)
 tr3.Processes.Default.Streams.stdout = Testers.ContainsExpression("Activity Timeout", "Request should fail with active timeout")
+
+if Condition.HasCurlFeature('http3'):
```

Comment:
This also needs to check HasATSFeature('http3').

Reply:
I am good with this.. the general question is if @zwoop is concerned with a test being skipped with no message that it was. I know he has a feature request in AuTest to have an option to force skipped test to be reported as an error